src/main/java/technology/tabula/CommandLineApp.java

```diff
@@ -216,75 +645,144 @@ private static OutputFormat whichOutputFormat(CommandLine line) throws ParseExce
-            List<Float> f = parseFloatList(optionValue.substring(startIndex));
-            if (f.size() != 4) {
-                throw new ParseException("area parameters must be top,left,bottom,right optionally preceded by %");
+            else{
+                System.out.println("ABSOLUTE ARGUMENT");
+                areaType.add(ABSOLUTE_AREA_CALCULATION_MODE);
+                sanitizedAreaArgs.add(areaArgs[index]);
             }
-            areaList.add(new Pair<Integer, Rectangle>(areaCalculationMode, new Rectangle(f.get(0), f.get(1), f.get(3) - f.get(1), f.get(2) - f.get(0))));
         }
-        return areaList;
+
+
+        System.out.println("Sanitized Options...");
+        System.out.println(sanitizedAreaArgs.toString());
+
+        //List<Float> f = parseFloatList(Arrays.asList(line.getOptionValues('a')).toString());
+
+        List<Float> f = parseFloatList(sanitizedAreaArgs.toString());
+
+        //throw new ParseException("area parameters must be top,left,bottom,right optionally preceded by %");
+
+        if((f.size()%4)!=0){
+            throw new ParseException("area parameters must be top,left,bottom,right");
+        }
+
+        ArrayList<technology.tabula.Pair<Integer,Rectangle>> pageAreas = new ArrayList<>();
+
+
+
+        for(Integer i=0; i<f.size(); i+=4){
+
+            pageAreas.add(new technology.tabula.Pair<Integer,Rectangle>(
+                    areaType.remove(0), new Rectangle(f.get(i),f.get(i+1),f.get(i+3)-f.get(i+1),f.get(i+2)-f.get(i))));
+        }
+
+        System.out.println("Specified Page Areas:");
+        System.out.println(pageAreas);
+
+        //return new Rectangle(f.get(0), f.get(1), f.get(3) - f.get(1), f.get(2) - f.get(0));
+        return pageAreas;
     }
 
-    private static List<Integer> whichPages(CommandLine line) throws ParseException {
-        String pagesOption = line.hasOption('p') ? line.getOptionValue('p') : "1";
-        return Utils.parsePagesOption(pagesOption);
+    private static List<List<Integer>> whichPages(CommandLine line) throws ParseException {
+        if(line.hasOption('p')){
+            String[] foo = Arrays.asList(line.getOptionValues('p')).toString().trim().replaceAll("[\\[\\]]","").split(",");
+
+            ArrayList<List<Integer>> allPagesWithAreas = new ArrayList<>();
+            for(String bar : foo){
+                allPagesWithAreas.add(Utils.parsePagesOption(bar));
+            }
+
+            return allPagesWithAreas;
+        }
+       else{
+            return new ArrayList<>();
+        }
+        //String pagesOption = line.hasOption('p') ? foo : "1";
+        //System.out.println(pagesOption);
+        //return Utils.parsePagesOption(pagesOption);
+
     }
 
     private static ExtractionMethod whichExtractionMethod(CommandLine line) {
-        // -r/--spreadsheet [deprecated; use -l] or -l/--lattice
-        if (line.hasOption('r') || line.hasOption('l')) {
+        // -r/--spreadsheet [deprecated, -r used for regex] or -l/--lattice
```

Comment:
Let's maintain backwards compatibility. Find another option name for regex search; `-r` is already taken by `--spreadsheet`